rhessys/tec/input_new_patch_mult.c

```diff
@@ -42,7 +42,7 @@
 #include "rhessys.h"
 #include "phys_constants.h"
 #define ONE 1.0
-
+#include "params.h"
 
  void  input_new_patch_mult(
```

Comment:
This code completely duplicates the logic for reading a single patch.  This means that when adding/changing a variable, it must be added/changed in multiple places, which is of course error prone.  We should take this opportunity to refactor the code to remove this kind of duplication.